Sources/EventSource/EventSource.swift

```diff
@@ -213,6 +214,7 @@ public extension EventSource {
                 
                 func sendErrorEvent(with error: Error) {
                     continuation.yield(.error(error))
+                    continuation.finish()
```

Comment:
Finishing the continuation here is redundant, since `close()` is always called after `sendErrorEvent(with:)`

Reply:
You are right. I will rm it.